lib/screens/oneOnOne/select_employee_view.dart

```diff
@@ -92,7 +92,7 @@ class _SelectEmployeeViewState extends State<SelectEmployeeView> {
                   maxRadius: 28.0,
                   foregroundImage: CachedNetworkImageProvider(employee?.avatarAttachmentUrl ?? ""),
                   child: Text(
-                    getInitials(employee?.name ?? "", 2),
+                    getInitials(employee?.name ?? "", 2).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.